packages/sync/package.json

```diff
@@ -44,7 +44,7 @@
     "typescript": "^3.1.6"
   },
   "peerDependencies": {
-    "@aerogear/core": ">2.0.0"
+    "@aerogear/core": ">2.0.1"
```

Comment:
Where is this version number coming from?

Reply:
Lowest supported version that will match this. Explained that in releng docs.